fairing/kubernetes/manager.py

```diff
@@ -24,13 +24,19 @@ def create_job(self, namespace, job):
     def create_tf_job(self, namespace, job):
         """Create the provided TFJob in the specified namespace"""
         api_instance = client.CustomObjectsApi()
-        return api_instance.create_namespaced_custom_object(
-            constants.TF_JOB_GROUP,
-            constants.TF_JOB_VERSION,
-            namespace,
-            constants.TF_JOB_PLURAL,
-            job
-        )
+        try:
+            return api_instance.create_namespaced_custom_object(
+                constants.TF_JOB_GROUP,
+                constants.TF_JOB_VERSION,
+                namespace,
+                constants.TF_JOB_PLURAL,
+                job
+            )
+        except client.rest.ApiException:
+            logger.error("Failed to create TFJob. Checked the cluster to ensure the "
+                         "TFJob version {} installed.".format(constants.TF_JOB_VERSION))
+        except Exception as e:
+            logger.error("Failed to create TFJob: {}".format(e))
```

Comment:
re throw the exception instead of catching it here.

Reply:
Updated, thanks